BitwardenShared/UI/Auth/IntroCarousel/IntroCarouselState.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+// MARK: - IntroCarouselState
+
+/// An object that defines the current state of a `IntroCarouselView`.
+///
+struct IntroCarouselState: Equatable {
+    // MARK: Types
+
+    /// A model representing the data to display on a single page in the carousel.
+    ///
+    struct CarouselPage: Equatable, Identifiable {
```

Comment:
🤔 Is there a reason the `CarouselPage` needs to be `Identifiable`? I was able to achieve the same effect with just an [Equatable enum](https://github.com/bitwarden/authenticator-ios/blob/main/AuthenticatorShared/UI/Platform/Tutorial/Tutorial/TutorialState.swift#L32) in the Authenticator app.

Reply:
It needed to be `Identifiable` to iterate over the array with `ForEach`. An enum could work too, I just liked the ability to define the list of pages as a list within the state.